### ISSUE TITLE
Incorrect/incomplete training loss formula in Paper section 3.3 equation 2?

### ISSUE THREAD

**Issue by Polaralia** [damo-vilab/AnyDoor#12]
Hi, there is a problem with equation (2) in section 3.3 of your paper? I read the paper and it says that you use training pairs for variation of the same object between frames. However, your equation 2 show a normal reconstruction loss for one frame. How do you use the paired training data?

**Comment by XavierCHEN34** [comment]
It is exactly a normal reconstruction loss. The paired frame is used for the condition  C  but not the input x
